tests/test_onnx_e2e.py

```
@@ -0,0 +1,393 @@
+    def __init__(self, fn):
+        super().__init__()
+        self._fn = fn
+
+    def forward(self, *inputs):
+        return self._fn(*inputs)
+
+
+def run_onnx(proto, inputs):
+    session = onnxruntime.InferenceSession(
+        proto.SerializeToString(), providers=["CPUExecutionProvider"])
+    feeds = {meta.name: np.asarray(data) for meta, data in zip(session.get_inputs(), inputs)}
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)